neo/Network/NodeLeader.py

```diff
@@ -203,6 +211,9 @@ def RemoveConnectedPeer(self, peer):
         if peer in self.Peers:
             self.Peers.remove(peer)
 
+    def onSetupConnectionErr(self, err):
+        print("On setup connectioin error! %s" % err)
```

Comment:
typo `connectioioioiooion`
I'm afraid this might spam prints in console when clients don't respond and we keep retrying. should this perhaps be a Log/Debug print instead?